apps/meteor/tests/e2e/file-upload.spec.ts

```diff
@@ -76,3 +76,27 @@ test.describe.serial('file-upload', () => {
 		await expect(poHomeChannel.content.btnModalConfirm).not.toBeVisible();
 	});
 });
+test.describe('file-upload-not-member', () => {
```

Comment:
Should we check how these behaves on Omni rooms?

Reply:
I don`t think so, they behave exactly the same as the other channels in this regard, the agent/manager cannot send messages in the room unless they are a part of it

Reply:
@MartinSchoeler indeed, not necessary!

Reply:
noice noice